src/Template/Element/Modules/thumb.twig

```diff
@@ -1,6 +1,12 @@
 {# thumb #}
 
-{% if object.type ==  'images' %}
+{# Use external provider thumbnail if available #}
+{% if object.attributes.provider_thumbnail %}
+
+        <img src={{ object.attributes.provider_thumbnail }} width="50" />
```

Comment:
At line 11 there's a `{% set width = 50 %}`, to have a `width` variable. I'd use it here as well... So we could move `{% set width = 50 %}` to line 2... and then change line 6 as follows:
```
<img src={{ object.attributes.provider_thumbnail }} width={{ width }} />
```